packages/frontend/amp/components/Epic.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import { headline, textSans } from '@guardian/pasteup/typography';
+import { palette } from '@guardian/pasteup/palette';
+
+const ampAccessConfig = {
+    authorization: 'http://localhost:3011/epic-api/auth',
+    pingback: 'http://localhost:3011/epic-api',
+    login: {
+        'sign-in':
+            'https://ampbyexample.com/components/amp-access/login?rid=READER_ID&url=CANONICAL_URL',
+        'sign-out': 'https://ampbyexample.com/components/amp-access/logout',
+    },
+    authorizationFallbackResponse: {
+        error: true,
+        loggedIn: false,
+        powerUser: false,
+    },
+};
+
+export const getAmpAccessScripts = () => {
+    return [
+        `<script async custom-element="amp-access" src="https://cdn.ampproject.org/v0/amp-access-0.1.js"></script>`,
+        `<script id="amp-access" type="application/json">${JSON.stringify(
+            ampAccessConfig,
+        )}</script>`,
+    ];
+};
+
+const url =
+    'https://interactive.guim.co.uk/docsdata/1fy0JolB1bf1IEFLHGHfUYWx-niad7vR9K954OpTOvjE.json';
+
+const container = css`
+    border-top: 0.0625rem solid ${palette.highlight.main};
+    background-color: ${palette.neutral[97]};
+    margin-top: 1.5rem;
+    padding: 0.25rem 0.3125rem 0.75rem;
+`;
+
+const heading = css`
```

Comment:
Are there possibility styles/components for this stuff already?

Reply:
Might be worth a look https://support-ui.gutools.co.uk/?selectedKind=Type&selectedStory=Text&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

Reply:
I'm not sure, but I'll hunt around and see what I find